notebooks/108-gpu-device/108-gpu-device.ipynb

```diff
@@ -0,0 +1,928 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    def completion_callback(infer_request: InferRequest, results) -&gt; None:
</pre><p>it can be dangerous to store outputs in list here, just because async api does not guarantee consequential execution, you just can draw detections on wrong frame later. Please consider to provide frame id for callback</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
Hi @eaidova , can you please explain how to implement this? The code from this notebook is based off the [classification_sample_async.py](https://github.com/openvinotoolkit/openvino/blob/master/samples/python/classification_sample_async/classification_sample_async.py) sample. The 115-async-api notebook doesn't use AsyncInferRequest (which I think is the preferred way to implement asynchronous inferencing), and it's hard to understand how the code works in that one.

Reply:
Nevermind!! Looks like @juliomorero got it figured out 😸